.readthedocs.yml

```diff
@@ -1,13 +1,26 @@
-# Don't build any extra formats
-formats:
-    - none
-conda:
-    file: ci/rtd_env.yml
+# .readthedocs.yml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Build documentation in the docs/ directory with Sphinx
+sphinx:
+  configuration: docs/conf.py
+
+# Do not build the docs in additional formats (e.g. PDF and ePub), use default
+formats: []
+
+build:
+    image: latest
+
+# Optionally set the version of Python and requirements required to build your docs
 python:
   version: 3.8
-  setup_py_install: true
   install:
     - method: pip
       path: .
-    - method: setuptools
-      path: package
+      extra_requirements:
+         - doc
+    - requirements: docs/requirements.txt
```

Comment:
hmmm, I'm a bit confused here... are we using the `[doc]` extra, or installing from `docs/requirements.txt`? Aren't they mutually exclusive? I prefer the former fwiw.

Reply:
Both. The requirements for building the docs with sphinx on any machine are in the `[doc]` extra. The requirements that are specific to the RTD build and hosting are in the `docs/requirements.txt`. I thought about putting those in the `[doc]` extra but I didn't like that because they aren't really required except on RTD.